### PR TITLE
Reset default text margins, Add updated spacing to information-container, scheme-container

### PR DESCRIPTION
**Pull request by Rody-Kirwan**
Reset native HTML text element margins and padding. 
Apply updated UX spacing where reasonable and applicable

### Description
This PR implements the first stage of UX update. `_base.scss` will be used to set default styles on native HTML elements. For the purposes of this change I have just reset margins and padding on all text elements to 0. To maintain consitency - margins and padding are dictated by wrapper elements.

Where applicable I have applied spacing specified in UX doc. This includes spacing in info-container, scheme-container, operations (unexpanded) models (unexpanded). I have purposefully omitted any updates within the expanded Operations and Models features as the proposed UX for these is drastically changed. 

Following discussion with @kyle.shokey I have also used "rem" as the standard unit of measurement for spacing. (font-sizes remain in px as this will be covered in upcoming PR)

### Motivation and Context
Swagger UI Facelift

### How Has This Been Tested?

Manually

### Screenshots (if appropriate):

Screen Shot 2019-04-26 at 16 03 34

Screen Shot 2019-04-26 at 16 06 38

Screen Shot 2019-04-26 at 16 07 22

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [X] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [X] are not breaking changes.

### Documentation
- [X] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [X] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by daniel-glynn — APPROVED** [review]

**Review by taraokelly — APPROVED** [review]

File: src/style/_base.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+input {
```
Comment (Rody-Kirwan) [inline src/style/_base.scss:1]: Setting default (16px) here as we are setting root font-size to 62.5% - `1rem => 10px`
